docs/pallas/grid_blockspec.md

```diff
@@ -0,0 +1,208 @@
+(pallas_grids_and_blockspecs)=
+
+# Grids and BlockSpecs
+
+(pallas_grid)=
+### `grid`, a.k.a. kernels in a loop
+
+When using {func}`jax.experimental.pallas.pallas_call` the kernel function
+is executed multiple times on different inputs, as specified via the `grid` argument
+to `pallas_call`. Conceptually:
+```python
+pl.pallas_call(some_kernel, grid=(n,))(...)
+```
+maps to
+```python
+for i in range(n):
+  # do HBM -> SRAM copies
+  some_kernel(...)
+  # do SRAM -> HBM copies
+```
+Grids can be generalized to be multi-dimensional, corresponding to nested
+loops. For example,
+
+```python
+pl.pallas_call(some_kernel, grid=(n, m))(...)
+```
+is equivalent to
+```python
+for i in range(n):
+  for j in range(m):
+    # do HBM -> SRAM copies
+    some_kernel(...)
+    # do SRAM -> HBM copies
```

Comment:
I think that the part about HBM SRAM was a good fit for the pipelining tutorial but is not relevant here. Perhaps remove the comments? And perhaps restate this example in the pipelining tutorial with the comments added?

Reply:
I removed these comments, but I do not want to redo the example in pipelining just for this.